packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -351,12 +351,12 @@ export namespace CommonCommands {
     });
 }
 
-export const supportCut = browser.isNative || document.queryCommandSupported('cut');
-export const supportCopy = browser.isNative || document.queryCommandSupported('copy');
+export const supportCut = environment.electron.is() || document.queryCommandSupported('cut');
+export const supportCopy = environment.electron.is() || document.queryCommandSupported('copy');
 // Chrome incorrectly returns true for document.queryCommandSupported('paste')
 // when the paste feature is available but the calling script has insufficient
 // privileges to actually perform the action
-export const supportPaste = browser.isNative || (!browser.isChrome && document.queryCommandSupported('paste'));
+export const supportPaste = environment.electron.is() || (!browser.isChrome && document.queryCommandSupported('paste'));
```

Comment:
I'm not sure that we need the special Chrome handling anymore, see https://github.com/w3c/editing/issues/167, https://bugs.chromium.org/p/chromium/issues/detail?id=757140#c4, and https://chromium.googlesource.com/chromium/src.git/+/29e7204c4d91b754fe8df8e264695a695a5ab974.